frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue

```diff
@@ -140,7 +137,7 @@ export default {
         feedbackType: "ERROR",
       },
       messageNotLabels:
-        "To create new rules, you need al least two labels. We highly recommended starting by annotating some records with these labels.",
+        "To create new rules, you need al least two labels. <br />We highly recommended starting by annotating some records with these labels.",
```

Comment:
Change suggestion (line 140):
"We highly recommend starting by annotating some records with these labels."